src/client/entity-editor/common/validation-label.tsx

```diff
@@ -88,15 +89,16 @@ function ValidationLabel({
 	children,
 	empty,
 	error,
+	isUf,
 	errorMessage,
 	warn,
 	warnMessage
 }: Props) {
 	const warnElement = (warn && !empty && !error) &&
 		<span className={contextualColor(empty, error, warn)}> {warnMessage} </span>;
-	const errorElement = errorMessage &&
+	const errorElement = errorMessage && error &&
 		<span className={contextualColor(empty, error, warn)}> {errorMessage} </span>;
-	const iconElement = icon(empty, error, warn) &&
+	const iconElement = !isUf && icon(empty, error, warn) &&
```

Comment:
It is only for tabs validation, icon on tab titles weren't looking great thus thought it would be better to remove them.

![uf-tabs-validation](https://user-images.githubusercontent.com/62337564/183820525-a218faf6-9918-4ce5-9ab4-b771afe3843b.png)

Reply:
I see. Then perhaps the `isUF` boolean should be renamed. Perhaps a more generic `hideIcon` (or `showIcon` defaulting to `true`)?